### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by MarcelBruckner**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -100,10 +100,8 @@
     "scoped_allocator": "cpp",
     "shared_mutex": "cpp"
   },
-  "workbench.colorCustomizations": {
```
Comment (marcelbrucker) [inline .vscode/settings.json:103]: Isn't this block from fortiss and should be kept?

File: bark_ml/library_wrappers/lib_tf2rl/runners/gail_runner.py [hunk 2]
```diff
@@ -6,58 +10,51 @@
...
     instantiated by self.GetTrainer.
     """
     self._trainer()
 
-  
   def GetTrainer(self):
     """Creates an IRLtrainer instance."""
     policy = self._agent.generator   # the agent's generator network, so in our case the DDPG agent
-    irl = self._agent.discriminator  # the agent's discriminator network so in our case the GAIL network
+    # the agent's discriminator network so in our case the GAIL network
```
Comment (marcelbrucker) [inline bark_ml/library_wrappers/lib_tf2rl/runners/gail_runner.py:52]: Maybe not so nice because it's inconsistent.

File: examples/gail.py [hunk 3]
```diff
@@ -75,23 +76,26 @@ def run_configuration(argv):
   expert_trajectories = None
   if FLAGS.mode != 'visualize':
     expert_trajectories, avg_trajectory_length, num_trajectories = load_expert_trajectories(params['ML']['ExpertTrajectories']['expert_path_dir'],
-      normalize_features=params["ML"]["Settings"]["NormalizeFeatures"],
-      env=env, # the unwrapped env has to be used, since that contains the unnormalized spaces.
-      subset_size=params['ML']['ExpertTrajectories']['subset_size']
-      ) 
+                                                                                            normalize_features=params["ML"][
```
Comment (marcelbrucker) [inline examples/gail.py:79]: This looks odd to me.

File: utils/tests/test_generate_launch_configuration.py [hunk 4]
```diff
@@ -1,19 +1,20 @@
 from generate_launch_configuration import generate
```
Comment (marcelbrucker) [inline utils/tests/test_generate_launch_configuration.py:1]: Suggestion to place 'test' at the end of the file name for consistency.

File: .vscode/settings.json (hunk 1 above)
Comment (MarcelBruckner) [reply .vscode/settings.json:103]: Its from one of my plugins.

File: bark_ml/library_wrappers/lib_tf2rl/runners/gail_runner.py (hunk 2 above)
Comment (MarcelBruckner) [reply bark_ml/library_wrappers/lib_tf2rl/runners/gail_runner.py:52]: I see, but this is the standard. I will now merge it as we can train on the dataset now and this should be our focus.

File: examples/gail.py (hunk 3 above)
Comment (MarcelBruckner) [reply examples/gail.py:79]: Obsolete now

**Pull request merged**